pkgs/development/php-packages/relay/default.nix

```diff
@@ -0,0 +1,99 @@
+{ stdenv
+, lib
+, php
+, openssl
+, openssl_1_1
+, zstd
+, lz4
+, autoPatchelfHook
+}:
+
+let
+  version = "0.6.3";
+  system = stdenv.hostPlatform.system;
+  phpVersion = lib.versions.majorMinor php.version;
+  variation = {
```

Comment:
To allow automatic updating, you might want to add an `updateScript`. Example: https://github.com/NixOS/nixpkgs/blob/c1871b41434c2c2deb730a771a49480978cdf5a6/pkgs/development/tools/misc/blackfire/php-probe.nix#L104-L162

Reply:
It would be also useful for regenerating the hashes to match `fetchurl` but it is up to you.